src/pygeosx/PyGroup.cpp

```diff
@@ -250,14 +254,31 @@ static PyObject * PyGroup_getWrapper( PyGroup * const self, PyObject * const arg
     return nullptr;
   }
 
-  string groupPath, wrapperName;
-  std::tie( groupPath, wrapperName ) = splitPath( path );
-
   try
   {
-    dataRepository::Group & group = self->group->getGroupByPath( groupPath );
-    dataRepository::WrapperBase & wrapper = group.getWrapperBase( wrapperName );
-    return createNewPyWrapper( wrapper );
+    geosx::ProblemManager * pbManager = static_cast< geosx::ProblemManager * >(self->group);
```

Comment:
So this is a function that's defined on `PyGroup` but is only valid if it's called on the `PyGroup` wrapping the problem manager?

Reply:
In the second version of creating a PySolver, this was kind of an assumption that it should be done from a ProblemManager PyGroup since I found bizarre to create a Solver from a Group that would not be related to Solver